new-examples/mysql-batch-execute-operation/mysql_batch_execute_operation.metatags

```diff
@@ -0,0 +1,2 @@
+description: This BBE on how to use the MySQL client to execute a batch of DDL/DML operations in Ballerina.
```

Comment:
```suggestion
description: This BBE demonstrates how to use the MySQL client to execute a batch of DDL/DML operations in Ballerina.
```
Please correct all occurrences.

Reply:
Corrected in all occurrences https://github.com/ballerina-platform/ballerina-distribution/pull/1751/commits/59d9b00bc36d17de3e5a905afec1522691dc612c

Reply:
Corrected in all occurrences https://github.com/ballerina-platform/ballerina-distribution/pull/1751/commits/59d9b00bc36d17de3e5a905afec1522691dc612c